<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="DESKTOP-90T0URJ" failures="1" tests="2" name="Default test(failed)" time="0.187" errors="0" timestamp="2022-10-08T01:01:25 EEST">
  <testcase classname="Demo.TestNGRetryFailedDemo" name="test2" time="0.027"/>
  <testcase classname="Demo.TestNGRetryFailedDemo" name="test3" time="0.04">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at Demo.TestNGRetryFailedDemo.test3(TestNGRetryFailedDemo.java:19)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- test3 -->
  <testcase classname="Demo.TestNGRetryFailedDemo" name="test1" time="0.0">
    <ignored/>
  </testcase> <!-- test1 -->
</testsuite> <!-- Default test(failed) -->
